lib/Graph/Nodes.cpp

```diff
@@ -180,7 +180,7 @@ static bool verifyFullyConnected(NodeValue src, NodeValue weights,
 static bool verifyPool(NodeValue src, NodeValue dest,
                        llvm::ArrayRef<unsigned_t> kernels,
                        llvm::ArrayRef<unsigned_t> strides,
-                       llvm::ArrayRef<unsigned_t> pads) {
+                       llvm::ArrayRef<unsigned_t> pads, bool isAvgPool = true) {
```

Comment:
i have a dejavu that we've already changed this :D

Reply:
Yes, in loading quantized model :P  Since this PR also need this part, I added it as well. Based on which one is merged first, the rest will be rebased :)